src/lib/learn-client/types.ts

```diff
@@ -261,5 +261,25 @@ export enum CompanyLogoOption {
 export function themeIsProduct(
   theme: ThemeOption | ProductOption
 ): theme is ProductOption {
-  return Object.values(ProductOption).includes(theme as ProductOption)
+  return isProductOption(theme)
+}
+/**
+ * Type guard to determine if a string is a ThemeOption or ProductOption
```

Comment:
I've added these type guards here in `lib` since they felt very generic. I'm using `isThemeOrProduct` in `detect-and-reformat-learn-url`.

Reply:
lol reading these function names just makes me think we _really_ need to refactor / address the product / theme weirdness with Learn 😆